From pavel@ucw.cz  Thu Nov 19 14:50:34 2009
From: Pavel Machek <pavel@ucw.cz>
Date: Wed, 28 Oct 2009 23:48:18 +0100
Subject: Staging: dream: add TODO file
To: Greg KH <greg@kroah.com>, Arve Hj?nnev?g <arve@android.com>, Brian Swetland <swetland@google.com>
Message-ID: <20091028224817.GA13987@elf.ucw.cz>
Content-Disposition: inline


This adds TODO list. It is probably incomplete, as many parts were not
reviewed by the upstream maintainers, yet.

Signed-off-by: Pavel Machek <pavel@ucw.cz>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/staging/dream/TODO |   14 ++++++++++++++
 1 file changed, 14 insertions(+)

--- /dev/null
+++ b/drivers/staging/dream/TODO
@@ -0,0 +1,14 @@
+* remove support for wakelocks since those are not in mainline
+
+* camera driver uses old V4L API
+
+* coding style in some places is lacking
+
+* gpio_input.c has some features matrix_keypad lacks. They should be
+merged to gpio_input, with gpio_input.c removed
+
+* pmem provides interface for userspace. Needs to be reviewed at least.
+
+* it is probably possible to simplify touchscreen driver using threaded_irq's.
+
+* touchscreen driver should be switched to oficial multitouch API
